src/js/control-bar/volume-control/volume-bar.js

```diff
@@ -106,7 +106,7 @@ class VolumeBar extends Slider {
    */
   updateARIAAttributes(event) {
     // Current value of volume bar as a percentage
-    const volume = (this.player_.volume() * 100).toFixed(2);
+    const volume = (this.player_.volume() * 100).toString();
```

Comment:
I think we'd still want Math.round or Math.floor to make sure we are getting an integer.